packages/compass-aggregations/src/components/stage-toolbar/index.tsx

```diff
@@ -97,6 +102,8 @@ const DISABLED_TEXT = 'Stage disabled. Results not passed in the pipeline.';
 const COLLAPSED_TEXT =
   'A sample of the aggregated results from this stage will be shown below.';
 
+const GUIDE_CUE_DELAY = 700;
```

Comment:
what is the delay for?

Reply:
I added this delay to give some time for the content to load (Preview documents) and not show Guide Cue immediately.

Reply:
did we consider other options too? what is the issue without the delay?